rules/sinks/third_parties/sdk/google/javascript.yaml

```diff
@@ -375,4 +375,13 @@ sinks:
       - "cloud.google.com/recaptcha-enterprise"
     patterns:
       - "react-google-recaptcha|vue-recaptcha|@google-cloud\\/recaptcha-enterprise|react-google-recaptcha-v3"
+    tags:
+
+  - id: ThirdParties.SDK.Pixel.Google.Analytics
+    name: Google Analytics
+    domains:
+      - "analytics.google.com"
+    filterProperty: "code"
+    patterns:
+      - "((\\$)?(window|w|this)[.])?(ga|datalayer)([.]push)?[(].*"
```

Comment:
check this regex once - this will make sure we miss lesser things:

`(.*(window|w|this)[.])?(ga|datalayer)([.]push)?[(].*`